go/logger/interfaces.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package logger
+
+import (
+	"golang.org/x/net/context"
+)
+
+type FLogger interface {
+	CriticalfLogger
+	DebugfLogger
+	ErrorfLogger
+	FatalfLogger
+	InfofLogger
+	NoticefLogger
+	WarningfLogger
+}
+
+type ContextFLogger interface {
+	CCriticalfLogger
```

Comment:
Are these single method interfaces necessary.

Also anything that isn't exportable should be lowercase.
And everything exportable should have comments.